# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1324465683 -3600
#      Wed Dec 21 12:08:03 2011 +0100
# Branch gui
# Node ID 995c69c217e579a964757e29e1746985196ed418
# Parent  c72ad9f5fa2ead0782627e4239bf4260093b76d8
Fixed bug in *.pro for compiling on Ubuntu.

diff --git a/gui/octave-gui.pro b/gui/octave-gui.pro
--- a/gui/octave-gui.pro
+++ b/gui/octave-gui.pro
@@ -47,17 +47,17 @@ mac {
 
 QMAKE_LIBDIR        += $$system(octave-config -p OCTLIBDIR)
 
 mac {
     LFLAGS += -L/opt/local-native/lib
 }
 
 unix {
-    QMAKE_LFLAGS +=  -lutil
+    LIBS +=  -lutil
 }
 
 win32-g++ {
     QMAKE_LFLAGS += --enable-auto-import
 }
 
 win32-msvc* {
     DEFINES += QSCINTILLA_DLL
